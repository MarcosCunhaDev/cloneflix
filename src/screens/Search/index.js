import React from "react";
import { View, Text } from "react-native";
import ContainerDefault from "../../components/Container";

function SearchScreen() {
  return (
    <ContainerDefault>
      <Text>Options Screen</Text>
    </ContainerDefault>
  );
}

export default SearchScreen;
