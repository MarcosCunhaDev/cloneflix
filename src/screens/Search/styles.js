import styled from "styled-components/native";
import { colors } from "../../config/const";

export const BoxResults = styled.View`
  flex: 1;
  flex-direction: row;
  flex-wrap: wrap;
  justify-content: flex-start;
  align-items: center;
  margin-top: 10px;
  margin-right: 16px;
  border-width: 1px;
  border-color: #fff;
  border-radius: 5px;
`;
export const Label = styled.Text`
  color: ${colors.secondaryColor};
`;
