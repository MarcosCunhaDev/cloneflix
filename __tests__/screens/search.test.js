import React from "react";
import renderer from "react-test-renderer";

import Search from "../../src/screens/Search";

describe("<Search />", () => {
  it("has 1 child", () => {
    const tree = renderer.create(<Search />).toJSON();
    expect(tree.children.length).toBe(1);
  });
});

it("renders correctly", () => {
  const tree = renderer.create(<Search />).toJSON();
  expect(tree).toMatchSnapshot();
});
