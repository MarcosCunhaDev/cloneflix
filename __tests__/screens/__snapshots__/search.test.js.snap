// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#121921",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "flex-start",
        "paddingLeft": 16,
        "paddingTop": 40,
      },
    ]
  }
>
  <RCTScrollView
    style={
      Object {
        "flex": 1,
        "width": "100%",
      }
    }
  >
    <View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderColor": "#fff",
              "borderRadius": 5,
              "borderWidth": 1,
              "flexDirection": "row",
              "height": 40,
              "marginRight": 16,
              "paddingHorizontal": 10,
            },
          ]
        }
      >
        <TextInput
          allowFontScaling={true}
          editable={true}
          maxLength={100}
          numberOfLines={1}
          onChangeText={[Function]}
          placeholder="Pesquisar Filme"
          placeholderTextColor="lightgrey"
          rejectResponderTermination={true}
          style={
            Object {
              "color": "white",
              "flex": 1,
              "fontSize": 16,
              "marginRight": 5,
            }
          }
          underlineColorAndroid="transparent"
          value=""
        />
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "height": 30,
                  "justifyContent": "center",
                  "width": 30,
                },
              ]
            }
          >
            <Text />
          </View>
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 5,
              "flexBasis": 0,
              "flexDirection": "row",
              "flexGrow": 1,
              "flexShrink": 1,
              "flexWrap": "wrap",
              "height": "100%",
              "justifyContent": "center",
              "marginRight": 16,
              "marginTop": 10,
              "paddingHorizontal": 20,
              "width": "100%",
            },
          ]
        }
      >
        <Text
          style={
            Object {
              "alignSelf": "center",
              "color": "#fff",
              "fontSize": 20,
              "textAlignVertical": "center",
            }
          }
        >
          Digite um nome para busca.
        </Text>
      </View>
    </View>
  </RCTScrollView>
</RCTSafeAreaView>
`;
